Specs/Scene/LabelCollectionSpec.js

```diff
@@ -875,16 +948,23 @@ defineSuite([
                     continue;
                 }
 
-                width += billboard.width;
-                height = Math.max(height, billboard.height);
+                var glyphWidth = billboard.width * scale;
```

Comment:
Is it possible to write a more coarse-grained expectation so we don't have to duplicate parts of the implementation in the tests?  Even if the expectation isn't as precise, it is probably more useful and robust across platforms.